lib/src/client/call.dart

```diff
@@ -239,12 +240,15 @@ class ClientCall<Q, R> implements Response {
       // Only received a header frame and no data frames, so the header
       // should contain "trailers" as well (Trailers-Only).
       _trailers.complete(_headerMetadata);
-      final status = _headerMetadata['grpc-status'];
+      _headerMetadata.remove(":status");
+      final status = _headerMetadata.remove('grpc-status');
```

Comment:
Why do you remove the header here and above?

Reply:
I tried to match what grpc-java returns when it calls a trailers in the same situation. I can just return a _headerMetadata.